master/buildbot/steps/artifact.py

```diff
@@ -135,7 +135,7 @@ def createSummary(self, log):
                 m = foundregex.search(l)
                 if (m):
                     artifactURL = self.artifactServerURL + "/" + self.artifactPath + "/" + a
```

Comment:
It's not your change but maybe safer here concatenate with urlparse.urljoin, otherwise can be a potential issue if someone decides to add a slash to the path.